### PR TITLE
Implement driver.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: src/r46ca01.toit [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  static BAUD_RATE_4800_ ::= 2
+  static BAUD_RATE_9600_ ::= 3
+  static BAUD_RATE_19200_ ::= 4
+
+  registers_/modbus.HoldingRegisters
+
+  /**
+  Creates a new R46CA01 driver.
+
+  The given Modbus $station must be a R46CA01 device.
```
Comment (erikcorry) [inline src/r46ca01.toit]:
```suggestion
  The given Modbus $station must be an R46CA01 device.
```

File: src/r46ca01.toit [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  */
+  read_correction --raw/bool -> int:
+    if not raw: throw "INVALID_ARGUMENT"
+    return read_ CORRECTION_ADDRESS_
+
+  /**
+  Sets the correction value.
+
+  The reported temperature is adjusted by this value. A positive value
+    increases the temperature, and a negative value decreases it.
```
Comment (erikcorry) [inline src/r46ca01.toit:102]: Is it stored non-volatile, or do you have to do this every time?

File: src/r46ca01.toit [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  */
+  set_correction value/float:
+    set_correction --raw (value * 10).to_int
+
+  /**
+  Sets the correction value.
+
+  Writes the given $value as raw value to the sensor.
+  Each unit corresponds to 0.1 degrees Celsius. A positive value
+    increases the temperature, and a negative value decreases it.
```
Comment (erikcorry) [inline src/r46ca01.toit]:
```suggestion
    increases the reported temperature, and a negative value decreases it.
```

**Review by erikcorry — APPROVED** [review]

File: src/r46ca01.toit (hunk 2 above)
Comment (floitsch) [reply src/r46ca01.toit:102]: non-volatile.
I updated the comment.
done.

**Pull request merged**